### ISSUE TITLE
Enhancement for separation of Wood & Tree types

### ISSUE THREAD

**Issue by Lylythii** [TerraFirmaCraft-The-Final-Frontier/TFC-TFF#9]
Some unique Tree types are in need of unique log blocks without registering a unique Wood type.
Eg: Different species of Hevea trees exist, which would have their own unique purposes (Tree tapping to produce different qualities of latex per tree species.), but only need one kind of hevea wood for blocks/items/etc.

Contrary to that, some trees do not need a unique log type. (This is already possible.)
Eg: Flowering Oak trees that do not have a "flowering oak" log.

Log blocks might need to be separated from both Wood & Tree types, & only be registered if one or both exist.

Creating unique woods without trees works flawlessly. The inverse doesn't work as well as it could.

**Comment by Tralomine** [comment]
two different logs of the same wood type don't really make much sense, if the logs are different then what you do with them is different, from a builder's point of view I would hate to have a new log texture but not a new plank with it, they can have very similar properties and textures, but if the logs are different in my opinion they are different woods
for tree taping different heveas, would taping a single separated log block work? if not, it can look for the leaves to know which trees it's taping from

and for a wood used for multiple trees yes that was one of the things I had in mind when doing the refactor

**Comment by Lylythii** [comment]
> two different logs of the same wood type don't really make much sense, if the logs are different then what you do with them is different
This is true, & precisely why I'm pointing out the problem. The logs provide a unique function (& not just in-world tree tapping, this was merely an example. Consider Ic² Extractors extracting latex from hevea logs for example.)

> from a builder's point of view I would hate to have a new log texture but not a new plank with it, they can have very similar properties and textures, but if the logs are different in my opinion they are different woods
This is scope creep. Something I've been trying to combat more as of late.
If I'm being completely honest I don't care enough about builders considering the textures of planks to sacrifice actual gameplay behaviours.
Aesthetics being integrated into builds should be opportunistic & creative.

> for tree taping different heveas, would taping a single separated log block work? if not, it can look for the leaves to know which trees it's taping from
This disregards that leaves are not what are being tapped, & that trimming a trees leaves shouldn't suddenly make the logs untappable.
It would likely not respect DramaticTrees trees either adding an unnecessary complexity like such.

> and for a wood used for multiple trees yes that was one of the things I had in mind when doing the refactor
Glad that worked out then.

**Comment by Tralomine** [comment]
I don't really get what you want, like, a log that when turned into planks gives the same planks as another one?

**Comment by Lylythii** [comment]
Non-issue for the meantime. Will resolve later.
